File: P1/Napakalaki/src/napakalaki/Monster.java

package napakalaki;

/**
 *
 * @author Francisco Fernández
 */
public class Monster {
    private String nombre;
    private int combatLevel;
    BadConsequence bc;
    Prize pc;
    
    public Monster(String name, int level, BadConsequence bc, Prize price){
        this.nombre=name;
        this.combatLevel=level;
    }
    public String getNombre(){
        return this.nombre;
    }
    public int getCombatLevel(){
        return this.combatLevel;
    }
    public BadConsequence getBc(){
        return bc;
    }
    public Prize getPc(){
        return pc;
    }
    public String toString(){
        return "Nombre = " + nombre + "CombatLevel = " + Integer.toString(combatLevel);
    }
    
}

File: P1/NapakalakiRuby/pkg/NapakalakiRuby-0.0.1/lib/Monster.rb
class Monster
  
  attr_reader :name, :combatLevel, :bc, :pc #Consultor GET

  def initialize(name, combatLevel, bc, pc)
    @name = name
    @combatLevel = combatLevel
    @bc = bc
    @pc = pc
  end
  
  def to_s
    "Nombre: #{@name} \n CombatLevel: #{@combatLevel} \n BadConsequence: #{@bc} \n Prize: #{@pc}"
  end
  
end
